dev-db/redict/redict-7.3.0.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# N.B.: It is no clue in porting to Lua eclasses, as upstream have deviated
+# too far from vanilla Lua, adding their own APIs like lua_enablereadonlytable
+
+inherit edo multiprocessing systemd tmpfiles toolchain-funcs
+
+SRC_URI="https://codeberg.org/redict/redict/archive/${PV/_/-}.tar.gz -> ${P}.tar.gz"
```

Comment:
let's put var assignments in the same order as skel.ebuild (pkgcheck should warn on this nowadays)

Reply:
fixed! turns out my pkgcheck was outdated